src/platform/silabs/rs911x/wfx_sl_ble_init.h

```diff
@@ -42,7 +42,7 @@
 #include <rsi_ble.h>
 #include <rsi_ble_apis.h>
 #include <rsi_ble_common_config.h>
-#ifndef SIWX_917
+#if !(SI917 | EXP_BOARD)

```

Comment:
it was recommended to use SIWX_917  instead of SI917

Reply:
Done